src/Analyzers/InternalRoslynAnalyzerFinder.cs

```diff
@@ -17,12 +18,14 @@ internal class InternalRoslynAnalyzerFinder : IAnalyzerFinder
 
         public Task<ImmutableArray<DiagnosticAnalyzer>> FindAllAnalyzersAsync(ILogger logger, CancellationToken cancellationToken)
         {
-            throw new NotImplementedException();
+            // The `GetAnalyzersAsync` will be fixed in a forthcoming build of ExternalAccess.Format
+            return Task.FromResult(CodeStyleAnalyzers.GetAnalyzersAsync());
```

Comment:
because it isn't async. I got ahead of myself then got even more ahead of myself.